source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -58,6 +58,17 @@ content: |
                put("secretAccessKey", awsSecretAccessKey);
              }});
            }};
+     .. tab::
+        :tabid: nodejs
+        
+        .. code-block:: javascript
+
+          kmsProviders = {
+              aws: {
+                  accessKeyId: // your AWS accessKeyId,
+                  secretAccessKey: // your AWS secretAccessKey,
+              }
+          }
```

Comment:
nit: 10 spaces -> 11 spaces

Reply:
Technically this isn't valid JSON because the commas are commented out. That's probably fine since we want them to replace values but it may cause some unnecessary headaches. We also have a style that is (mostly) valid that accomplishes this:

```javascript
const kmsProviders = {
  aws: {
    accessKeyId: "<IAM User Access Key ID>",
    secretAccessKey: "<IAM User Secret Access Key>",
  }
}
```

Reply:
Fixed